apps/packages/takes_package_service.py

```diff
@@ -40,7 +40,7 @@ def get_take_package_id(self, item):
         :return: _id of the package or None
         """
         takes_package = [package.get(PACKAGE) for package in item.get(LINKED_IN_PACKAGES, [])
-                         if package.get(PACKAGE_TYPE)]
+                         if package.get(PACKAGE_TYPE) is TAKES_PACKAGE]
```

Comment:
woops was foiled by my equality test in console.

http://stackoverflow.com/questions/1504717/why-does-comparing-strings-in-python-using-either-or-is-sometimes-produce

I don't have the data, I can try to get it from PA if it's still in their dev instance. 

These were in the logs

[32m11:05:00 work.1 | [0mHTTP Exception 403 has been raised: Multiple takes found for item: urn:newsml:localhost:2016-05-18T12:52:30.494683:af7d8887-8e97-47e3-b1cd-b5555fd67ea8 level=ERROR process=Worker-2
[32m11:05:00 work.1 | [0m403: Multiple takes found for item: urn:newsml:localhost:2016-05-18T12:52:30.494683:af7d8887-8e97-47e3-b1cd-b5555fd67ea8 level=ERROR process=Worker-2